unit/test_get_dist_dirs_to_remove.py

```diff
@@ -38,6 +38,13 @@ def setUp(self):
         # create app dist dir
         create_dir(os.path.join(self.app_install_dir, '%s-0.0.1-0' % APP_NAME))
 
+        # create stateboard dist dir - it shouldn't be removed
+        create_dir(os.path.join(self.app_install_dir, '%s-stateboard' % APP_NAME))
+
+        # create instances dist dirs - it shouldn't be removed
+        for instance_name in ['i-1', 'i-2', 'i-3']:
```

Comment:
Why are you using exactly `i-1`, `i-2` e.t.c.? What about using `APP_NAME-instance-1`?